lib/msf/ui/console/command_dispatcher/creds.rb

```diff
@@ -312,18 +314,41 @@ def build_service_info(service)
     info
   end
 
+  def creds_fill(*args)
+    index = args[0].to_i
+    if (index == 0)
+      print_error("Invalid parameter, #{args}")
+      return
+    else
+      index -= 1
+    end
+    opts = {}
+    usernames = []
+    passwords = []
+    query = framework.db.creds(opts)
+    query.each do |core|
+      user = core.public ? core.public.username : ''
+      usernames << user
+      passwd = core.private ? core.private.data : ''
+      passwords << passwd
```

Comment:
this doesn't include domain, maybe this should be expanded to handle `SMBDomain` and pull that from the db as well?

Reply:
Should it be just `SMBDomain` or others like `JENKINSDOMAIN` also